### PR TITLE
Moshe card gallery item

### PR DESCRIPTION
**Pull request by mosheKerbel**
This PR adds a badge to the cardGalleryItem like in the [spec](https://app.zeplin.io/project/5c8f6f1f9f489505cac756dc/screen/5cc97116201cf7684b654ffd)

### CODE REVIEW

File: src/common.scss [hunk 1]
```diff
@@ -7,3 +7,6 @@
     @content
   }
 }
+
+$cardBorderRadius: 8px;
```
Comment (mosheKerbel) [inline src/common.scss]: i am using the same value both in card(https://github.com/wix/wix-style-react/blob/6d1dfe4bd0840c37a9865b3b2b37c990fe25e175/src/Card/Card.scss#L5) and in cardGalleryItem(https://github.com/wix/wix-style-react/blob/6d1dfe4bd0840c37a9865b3b2b37c990fe25e175/src/CardGalleryItem/CardGalleryItem.scss#L14)
so i extracted it in here, not sure this is the right place for it.

**Comment by mosheKerbel** [comment]
@NitayRabi

File: src/common.scss (hunk 1 above)
Comment (NitayRabi) [reply src/common.scss]: Better then using it hardcoded in both places IMO :)

File: src/CardGalleryItem/CardGalleryItem.js [hunk 2]
```diff
@@ -57,10 +58,29 @@ class CardGalleryItem extends React.Component {
...
+      badge && (
+        <div className={styles.badgeWrapper}>
+          <Badge
+            size="medium"
+            skin="standard"
+            type="solid"
+            uppercase
+            data-hook={'badge'}
+          >
+            {badge}
```
Comment (NitayRabi) [inline src/CardGalleryItem/CardGalleryItem.js]: Is badge text? node? maybe `children` \ `text` is a better paramName?

File: src/CardGalleryItem/CardGalleryItem.uni.driver.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+const getBadge = base => base.$('[data-hook="badge"]');
 const getPrimaryAction = base => base.$('[data-hook="primary-action"]');
 const getSecondaryAction = base => base.$('[data-hook="secondary-action"]');
 const getHoverComponent = base => base.$('[data-hook="hover-component"]');
 
 const cardGalleryItemDriverFactory = base => ({
   ...baseUniDriverFactory(base),
   getTitle: () => getTitle(base).text(),
+  getBadge: async () =>
+    (await getBadge(base).exists()) ? getBadge(base).text() : null,
```
Comment (NitayRabi) [inline src/CardGalleryItem/CardGalleryItem.uni.driver.js]: Maybe `getBadgeText`?

File: src/CardGalleryItem/test/CardGalleryItem.spec.js [hunk 4]
```diff
@@ -30,6 +30,18 @@ describe('CardGalleryItem', () => {
     expect(await driver.getSubtitle()).toBe('Subtitle');
   });
 
+  it('should not render badge', async () => {
```
Comment (NitayRabi) [inline src/CardGalleryItem/test/CardGalleryItem.spec.js]: `'should not render badge by default'`

File: src/CardGalleryItem/CardGalleryItem.js (hunk 2 above)
Comment (mosheKerbel) [reply src/CardGalleryItem/CardGalleryItem.js]: changing to badgeText

**Pull request merged**